### DISCUSSION TITLE
Can the -d (--finaldir) argument be an IP address

### DISCUSSION THREAD

**Discussion by ghost** [madMAx43v3r/chia-plotter#896] [Q&A]
Can the -d (--finaldir) argument be a remote machine IP address such as user@192.168.1.100:/mnt/hdd ?
Assuming SSH is set up.

**Comment by idog8818** [comment]
I mount a windows shared folder via
sudo mount -t cifs -w -o username=UInput ’\\ip\sharedfolder /mnt/shareddisk

you could do similar in Linux

**Comment by llamasoft** [comment]
Just use SSHFS and "mount" it as a local drive: https://www.digitalocean.com/community/tutorials/how-to-use-sshfs-to-mount-remote-file-systems-over-ssh

**Comment by ghost** [comment]
I'm already using SSHFS to mount the drive but there is some overhead with encryption wondered if I could shave a few minutes off transfer time. I'm going to assume that the answer to my question is no then unless I hear otherwise. Thanks for replying.

**Reply by altendky** [reply]
Generally speaking you can't write a file to an IP address.  Also, wouldn't ssh have encryption overhead similar to sshfs?  There are however other tools such as https://github.com/ericaltendorf/plotman/ and https://github.com/rjsears/chia_plot_manager that provide features for transferring plots.  There are also Swar and Harry Plotter, though I don't know what they provide for plot transfers.

I'm admittedly biased here, but I think it makes a lot of sense for the plotters themselves to just make a plot and stick it somewhere simple.  Other tools can handle the various scheduling and completed plot management activities without making each plotter author have to deal with them all.

**Comment by svennand** [comment]
I've done this for my 100TB setup. All transfered over network without issue.
Keep in mind you might get some stale shares if your writing to a disk you are farming on.

sudo nano /root/.smbcredentials (add username and password for smb on two lines)
sudo apt install -y cifs-utils.
sudo mount -t cifs -o rw,vers=3.0,credentials=/root/.smbcredentials //192.168.0.10/sharename/folder /media/shareX/
